test/Base_Tests/src/Semantic/Type_Refinement/Types.enso

```diff
@@ -0,0 +1,20 @@
+from project.Semantic.Type_Refinement.Hidden_Conversions import all
```

Comment:
private?

Reply:
Could be, although this is a `_Tests` project so I don't know if it needs to be.

It was meant to resemble our usecases (e.g. `Column & Integer` from other PR) - there too the types themselves are public, only the conversions are private.